datasource/src/main/resources/application.conf

```diff
@@ -14,4 +14,21 @@ database {
   liquibase {
     changeLogResourcePath = "liquibase/changelog/changelog-master.xml"
   }
+
+  mongo {
+    user = ${?CMWL_MONGO_USER}
+    user = "user"
```

Comment:
You don't need to specify the fields manually, they are already specified in the .env.example

Reply:
Repository tests use this file

Reply:
If you have ```user = ${?CMWL_MONGO_USER}``` why do you specify ```user = "user"``` after?